<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDemoClass</class>
 <widget class="QMainWindow" name="ControlDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
    border: 1px solid #dcdcdc;
}

QTabBar::tab {
    height: 40px;
    width: 140px;
    background: #f5f5f5;
    margin: 2px;
    border-radius: 4px;
    writing-mode: horizontal-tb;

}

QTabBar::tab:selected {
    background: #409EFF;
    color: white;
    font-weight: bold;
}

QTabBar::tab:hover {
    background: #e6f2ff;
}



</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Button</string>
       </attribute>
       <widget class="StateButton" name="stateButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="test_state_btn">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>测试state Button</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>231</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 大号白色按钮 */
QPushButton {
    background-color: white;
    color: #4F46E5; /* indigo-600 */
    padding: 24px 48px;
    font-size: 18px;
    border-radius: 16px;
    box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.25);
    min-height: 24px;
}

QPushButton:hover {
    background-color: #EEF2FF; /* indigo-50 */
    transform: scale(1.02);
    box-shadow: 0 20px 25px -5px rgba(0, 0, 0, 0.1),
                0 10px 10px -5px rgba(0, 0, 0, 0.04);
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>280</y>
          <width>231</width>
          <height>32</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #059669; /* emerald-600 */
    color: white;
    padding: 16px 32px;
    border-radius: 12px;
    box-shadow: 0 6px 0 0 #047857; /* emerald-700 */
    transition: all 0.2s;
}

QPushButton:hover {
    transform: translateY(3px);
    box-shadow: 0 3px 0 0 #047857;
}

QPushButton:pressed {
    transform: translateY(6px);
    box-shadow: 0 0 0 0 #047857;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StateButton</class>
   <extends>QPushButton</extends>
   <header>./Button/statebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ControlDemo.qrc"/>
 </resources>
 <connections/>
</ui>
